cdap-standalone/bin/cdap.bat

```diff
@@ -205,6 +211,7 @@ SET "CLASSPATH=%CDAP_HOME%\lib\*;%CDAP_HOME%\conf\;"
 CALL :CHECK_WINDOWS
 CALL :CHECK_JAVA
 CALL :CREATE_LOG_DIR

```

Comment:
I think it's fine to add to this PR, but it's not necessary and can be done in another PR. In other words, don't let that block you.